launch/launch/launch_introspector.py

```diff
@@ -100,7 +103,7 @@ def format_event_handler(event_handler: EventHandler) -> List[Text]:
     """Return a text representation of an event handler."""
     if hasattr(event_handler, 'describe'):
         # TODO(wjwwood): consider supporting mode complex descriptions of branching
-        description, entities = event_handler.describe()
+        description, entities = event_handler.describe()  # type: ignore
```

Comment:
mypy was complaining about that `EventHandler` doesn't have a `describe` method: I think that ignoring it is the appropriate thing to do here.